src/Kafka/CreateKafkaInstance/components/InstanceInfo.tsx

```diff
@@ -57,15 +57,15 @@ export const InstanceInfo: VoidFunctionComponent<
   const { t } = useTranslation();
 
   return (
-    <TextContent>
+    <TextContent data-testid={"instance-info"}>
       <Text component={TextVariants.h3}>
         {t("create-kafka-instance:instance_information")}
       </Text>
-      <TextList component={TextListVariants.dl}>
-        {!isLoading && (
-          <Grid sm={6} lg={12} hasGutter>
-            {isTrial && (
-              <GridItem>
+      {!isLoading && (
```

Comment:
Layout changed to avoid showing the skeletons inside a DataList, since that will create invalid HTML markup (divs can't live inside a dl)